.github/workflows/integration_test_app.yaml

```diff
@@ -0,0 +1,56 @@
+name: Integration tests for application
+
+on:
+  pull_request:
+    paths:
+      - 'app/**'
```

Comment:
only run on code change . disadvantage: required status checks cannot be included. We may change this in a follow-up.

Reply:
I think this might need to be expanded, e.g., to include the test directory

Reply:
You might also want to include the `.github` directory because the workflow should run if it is changed. Perhaps look over the rest of the repo for any similar cases, e.g., config files etc

Reply:
Talked to Javi. We decided to drop the conditional run for now, mainly because of the missing required status check, which would allow people to merge PR's without running the test, although required. And these tests seem to be a lot faster than before after dropping microk8s&juju.